operator/src/main/java/oracle/kubernetes/operator/helpers/DomainValidationSteps.java

```diff
@@ -127,6 +127,10 @@ public NextAction apply(Packet packet) {
     private String perLine(List<String> validationFailures) {
       return String.join(lineSeparator(), validationFailures);
     }
+
+    private String domainValidated(Packet packet) {
+      return "Validated " + DomainPresenceInfo.fromPacket(packet).orElse(null);
```

Comment:
It seems debug message is printing the contents of DomainPresenceInfo. Should this be just the domain name?

Reply:
Yes, I agree. It would be better just to print the domain name.